src/www/messages/en.json

```
@@ -42,7 +42,7 @@
   "non-system-vpn-warning-detail": "Most browsers connect automatically with Outline, some do not.",
   "non-system-vpn-warning-title": "Verify your browser connection",
   "tray-open-window": "Open",
-  "outline-plugin-error-admin-permissions": "Outline needs admin permissions in order to run. If you don't have admin permissions, please try again after restarting your Windows computer or ask your system administrator for help.",
+  "outline-plugin-error-admin-permissions": "Outline needs admin permissions in order to run. If you don't have admin permissions, please try again after restarting your computer or ask your system administrator for help.",
```

Outline doesn't really need to ask for admin permissions to run. I think something like this would be more accurate:
```suggestion
  "outline-plugin-error-admin-permissions": "Outline needs admin permissions in order to set up the Outline VPN. If you don't have admin permissions, please try again after restarting your computer or ask your system administrator for help.",
```